test/Libraries/Microsoft.Extensions.AI.Integration.Tests/ChatClientIntegrationTests.cs

```diff
@@ -819,7 +820,7 @@ private class Person
 
     private enum JobType
     {
-        Surgeon,
+        Wombat,
```

Comment:
`llama3.1` is absolutely obsessed with claiming that people are surgeons, even Taylor Swift. I had to remove that as one of the options otherwise it would always pick it.